src/components/Rezume/Rezume.js

```diff
@@ -29,14 +29,25 @@ const Rezume = ({ Resume }) => {
           Phone: <strong>{Phone}</strong>
         </p>
       </div>
+      {ProfilePicture.Set ? (
+        <img
+          src={ProfilePicture.ProfileImage}
+          className="img-thumbnail float-left profile-image"
+          alt="profile"
+        />
+      ) : null}
       <h2>
         <span>{NameArray.shift()}</span> {NameArray.join(" ")}
       </h2>
       <p className="location">{Location}</p>
-      <p className="summary">
-        <strong className="position">{CurrentPosition}</strong>
-        {Summary}
-      </p>
+      <p
+				className={`summary${
+					ProfilePicture.Set ? " summary-margin" : ""
+				}`}
+			>
+				<strong className="position">{CurrentPosition}</strong>
+				{Summary}
+			</p>
```

Comment:
Prettier?

Reply:
Great - I guess you just need to do one thing. Change Tabs to 2 spaces. 😊

Reply:
@sebinfrancis7 This seems fine. Just run through prettier, even if the other lines aren't your code, gets changed, it's alright, we'll tweak it later. 😊